lib_standard_app/io.c

```diff
@@ -202,6 +208,10 @@ WEAK int io_send_response_buffer(const buffer_t *rdata, uint16_t sw) {
     return ret;
 }
 
+WEAK int io_send_response_buffer(const buffer_t *rdata, uint16_t sw) {
+    return io_send_response_buffers(rdata, 1, sw);
+}
+
 WEAK int io_send_sw(uint16_t sw) {
     return io_send_response_buffer(NULL, sw);
 }
```

Comment:
I'm wondering if we should inline those and put them on io.h to save a few bytes of flash...

Reply:
maybe this overkilled

Reply:
After discussion, let do it, we have already do simiar stuff on crypto_helper.h